presto-spi/src/main/java/io/prestosql/spi/block/LazyBlock.java

```diff
@@ -254,6 +254,7 @@ public Block getBlock()
         return block;
     }
 
+    @Deprecated
     public void setBlock(Block block)
```

Comment:
Since we're already making a backwards incompatible change, let's just remove this method now.